Components/Windows/create-local-user/README.md

```diff
@@ -0,0 +1,42 @@
+# Creation of a local Windows user account with RDP login permissions
+
+This is a sample component that demonstrates how to create a local Windows user account without Administrator permissions and grant RDP login privileges to this account. This can be useful in situation where you build a jump host that you do not want to join into an Active Directory and where you do not want the users to have admin access, especially if you manage your configuration changes to the machines via an EC2 Image Builder pipeline.
```

Comment:
A few grammar improvements. Changes in bold.

* in **a** situation
* into an Active Directory **domain** and

Reply:
Hi Andrew,

thanks a lot, I fixed that.